### ISSUE TITLE
CoE Starter Kit November 2023 Latest - CoreComponents_4.16 or CoreComponents_4.17?

### ISSUE THREAD

**Issue by jcgds** [microsoft/coe-starter-kit#7088]
### Does this question already exist in our backlog?

- [X] I have checked and confirm this is a new question.

### What is your question?

Hello, I intend to update the COE in my tenant to resolve the issues mentioned below, however, when I download via the link: https://aka.ms/CoeStarterKitDownload the file: CenterofExcellenceCoreComponents_nanaged has version 4.16 instead of 4.17 as mentioned in the github. Please, should I download from github instead of the link: https://learn.microsoft.com/en-us/power-platform/guidance/coe/after-setup?

Issues:
#6815-[CoE Starter Kit - BUG] Environment Request Maker page not working

#6946-[CoE Starter Kit - Feature]: First Modern Dataverse Connector usage: Admin | Sync Template v4 (Desktop flows)

CoEStarterKit with link
![CoeStarterKitLink](https://github.com/microsoft/coe-starter-kit/assets/147111987/51be73cf-ed2c-4cf4-9685-de1698e5a55b)

CoEStarterKit with Github
![CoeStarterKitGithub](https://github.com/microsoft/coe-starter-kit/assets/147111987/75dd2dbd-9710-4cd0-b632-5f746d0682d8)

### What solution are you experiencing the issue with?

None

### What method are you using to get inventory and telemetry?

None

**Comment by Jenefer-Monroe** [comment]
Yes it looks like we forgot to update the download link. You can download from here: https://github.com/microsoft/coe-starter-kit/releases/tag/CoEStarterKit-November2023
At the bottom
![image](https://github.com/microsoft/coe-starter-kit/assets/89584709/e2259e12-4236-40a8-96e9-2bbf2cdd2310)

**Comment by Jenefer-Monroe** [comment]
closing out as no further action for starter kit team
